jenkins.sh

```diff
@@ -35,19 +35,19 @@ set_kubeconfig() {
   echo "Kubeconfig created successfully"
 }
 
-set_quay_login() {
-  echo "Creating docker login for quay.io"
+set_docker_login() {
+  echo "Creating docker login"
   mkdir /root/.docker
-  cp /root/.secrets/dockercfg /root/.docker/config.json
-  echo "Docker login for quay.io created successfully"
+  cp /root/.secrets/config.json /root/.docker/config.json
+  echo "Docker login created successfully"
```

Comment:
Before saying that, you could check whether it really has been successfully created.